geonode_mapstore_client/client/js/epics/gnsearch.js

```diff
@@ -225,9 +242,27 @@ export const gnsSelectResourceEpic = (action$, store) =>
                 );
         });
 
+export const getFeaturedResourcesEpic = (action$, {getState = () => {}}) =>
+    action$.ofType(LOCATION_CHANGE, UPDATE_FEATURED_RESOURCES)
+        .filter(({ payload }) => {
+            return payload ? payload.location?.pathname === '/' : getState().router?.location?.pathname === '/';
```

Comment:
see previous comment about location

Reply:
This is used to trigger fetching the resources on location change. Maybe I could dispatch the action to fetch the featured resources manually in the useEffect